engine/src/main/java/org/terasology/rendering/dag/nodes/ChunksRefractiveReflectiveNode.java

```diff
@@ -77,7 +82,7 @@ public void process() {
      * accommodate the rendering of the water surface from an underwater point of view.
      */
     private void preRenderSetupSceneReflectiveRefractive() {
-        sceneReflectiveRefractive = frameBuffersManager.getFBO("sceneReflectiveRefractive");
+        sceneReflectiveRefractive = dynamicFBM.getFBO(REFRACTIVE_REFLECTIVE_URN);
```

Comment:
I've been thinking. 

FBMs, especially the dynamic ones, are a bit more than simple collections given what they do to regenerate the FBOs as needed. _However_ from the point of view of a node they are basically simple maps that can notify subscribers of changes. 

So, apart from the fact that this type of line shouldn't probably exist within process() and should be moved to update(). How about we also change the FBM interface so that we can write:

`sceneReflectiveRefractive = dynamicFBOs.get(REFRACTIVE_REFLECTIVE);`

?